### DISCUSSION TITLE
I want to call web AIP, but no matter how I manipulate controlnet, it doesn't work, including preprocessing, and I can't get preprocessing results. I need help

### DISCUSSION THREAD

**Discussion by dragoncdj** [Mikubill/sd-webui-controlnet#1542] [General]
This is the preprocessed code
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/132640267/ccf07b1f-38fa-4592-9ce6-8e4a6fe0f136)
This is to generate an image through ControlNet, but it does not find that ControlNet works, this example is exactly the same as in the documentation, except that the module and model are replaced.
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/132640267/ad1896d9-4d86-48b0-8e09-849cbf354fb7)
Official documentation document address：https://github.com/Mikubill/sd-webui-controlnet/wiki/API

Please ask where the problem is handled or where it is not set！

**Comment by ramyma** [comment]
For the txt2img payload, try adding `mask: None` next to `input_image`.

**Reply by 3DsWorks2022** [reply]
不是你一个人遇到这个问题了，很多人遇到问题了，搞不懂为什么把controlnet/txt2img 和 controlnet/img2img的 接口删掉，导致原先我们编写的代码全部报废了，报废也就算了，更换成 sdapi/v1/txt2img 也不生效，从昨天开始搞到今天毫无进展。

**Reply by dragoncdj** [reply]
知道问题是在于MLSD不生效
这个老哥也是这问题
https://github.com/Mikubill/sd-webui-controlnet/issues/1501

**Comment by ljleb** [comment]
What do you mean exactly by "it does not work"? Does the result look bad? Do you get an error message? Does the API just silently return nothing?

**Reply by dragoncdj** [reply]
It returns content, but it doesn't return it as expected. Later, I looked at the problem raised by others, and I knew that MLSD preprocessing would not work. The following friend is the problem.
Question link：https://github.com/Mikubill/sd-webui-controlnet/issues/1501

**Comment by lllyasviel** [comment]
added smart default value in 1.1.218 https://github.com/Mikubill/sd-webui-controlnet/pull/1565

**Reply by dragoncdj** [reply]
Thank you so much
